### ISSUE TITLE
generated backup file not valid

### ISSUE THREAD

**Issue by jtbayly** [ovidiul/XCloner-Wordpress#16]
I've got a 2GB backup file, that upon extracting, fails with the following error: "Truncated tar archive"

I'm not sure if it was caused by the host limiting file sizes or something, but regardless, this problem should throw an error, not fail silently.

**Comment by dichvuhuuich** [comment]
Do you try go to admin.php?page=xcloner_settings_page&tab=system_options and in "Split Backup Archive Limit (MB)" set to number your host limit.

**Comment by ovidiul** [comment]
jtbayly: Have you checked if the downloaded backup archive has the same size as the created one from the server?

**Comment by jtbayly** [comment]
@dichvuhuuich I saw that option and assumed it would fix my problem. But my concern is not that I be able to figure out a way to make the backup work. My concern is that I be able to *trust* the backup software to create a valid backup and let me know if it is unable to. That is why this is a big deal.

**Comment by jtbayly** [comment]
@ovidiul I believe I did at the time, but I no longer have the files to verify (or the time to go retest it at the moment). I'm sure the file sizes matched and were right at 2GB. I'm less certain I checked the file sizes down to the byte. 

I downloaded the backup file first from the browser. Then, when it failed to expand, I downloaded it again via an SFTP program. I had the same problem with both copies.
